.travis.yml

```diff
@@ -34,38 +43,21 @@ matrix:
       compiler: gcc-4.9
   fast_finish: true
 before_install:
-  # Pins the version of the java package installed on the Travis VMs
-  # and avoids a lengthy upgrade process for them.
-  - if [ $TRAVIS_OS_NAME = linux ]; then
-      sudo apt-mark hold oracle-java7-installer oracle-java8-installer;
-      sudo apt-get update;
-    elif [ $TRAVIS_OS_NAME = osx ]; then
+  - if [ $TRAVIS_OS_NAME = osx ]; then
       brew update;
     fi
+  - pip install --user -q --upgrade pip
 install:
-  - if [ $TRAVIS_OS_NAME = linux ]; then
-      sudo apt-get install xclip gdb;
-    elif [ $TRAVIS_OS_NAME = osx ]; then
+  - if [ $TRAVIS_OS_NAME = osx ]; then
       brew install gettext;
     fi
-before_script:
-  # Adds user to a dummy group.
-  # That allows to test changing the group of the file by `os_fchown`.
-  # Need xvfb for running some tests with xclip
-  - if [ $TRAVIS_OS_NAME = linux ]; then
-      sudo groupadd chown_test;
-      sudo usermod -a -G chown_test $USER;
-      export DISPLAY=:99.0;
-      sh -e /etc/init.d/xvfb start;
-    elif [ $TRAVIS_OS_NAME = osx ]; then
-      sudo dscl . -create /Groups/chown_test;
-      sudo dscl . -append /Groups/chown_test GroupMembership $USER;
-    fi
+  - pip install --user -q cpp-coveralls neovim
```

Comment:
Looks like you need to adjust the `$PATH`:

> /home/travis/build/neovim/neovim/.ci/gcc.sh: 37: /home/travis/build/neovim/neovim/.ci/gcc.sh: coveralls: not found
coveralls upload failed.

I think `~/.local/bin/` is where the executable ends up.